applications/crossbar/test/cb_alerts_tests.erl

```diff
@@ -0,0 +1,286 @@
+    [RespObj1] = cb_context:resp_data(Context5),
+    %% topup succeeds
+    meck:expect('kz_services_topup', maybe_topup, fun(_, _) -> MaybeTopupResult end),
+    Context6 = cb_alerts:check_low_balance(Context),
+
+    %% Unload mecked modules
+    lists:foreach(fun(M) -> meck:unload(M) end, ToMeck),
+
+    [{"If getting account's available units fails with 'timeout' 3 times in a row the context should not change"
+     ,?_assertEqual(Context1, Context)
+     }
+    ,{"If getting account's available units fails the context should not change"
+     ,?_assertEqual(Context2, Context)
+     }
+    ,{"If available dollars > Threshold dollars the context should not change"
+     ,?_assertEqual(Context3, Context)
+     }
+    ,{"When available dolars = Threshold dollars return alert"
+     ,?_assertEqual('true', kz_json:are_equal(F(ThresholdUSD, ThresholdUSD), RespObj))
+     }
+    ,{"When available dolars < Threshold dollars return alert"
+     ,?_assertEqual('true', kz_json:are_equal(F(AD, ThresholdUSD), RespObj1))
+     }
+    ,{"When available dolars < Threshold dollars but topup succeeds the context should not change"
+     ,?_assertEqual(Context6, Context)
+     }
+    ].
+
+%%%=======================================================================================
+%%% Internal (helpers)
+%%%=======================================================================================
+
+%%--------------------------------------------------------------------
+%% @doc
+%% @end
+%%--------------------------------------------------------------------
+-spec account_active_ports() -> {'ok', kz_json:objects()}.
+account_active_ports() ->
+    {'ok', [example_port_request(), scheduled_port_request(), rejected_port_request()]}.
+
+-spec scheduled_port_request() -> kz_json:object().
+scheduled_port_request() ->
+    kz_json:set_value(<<"pvt_port_state">>, <<"scheduled">>, example_port_request()).
+
+-spec rejected_port_request() -> kz_json:object().
+rejected_port_request() ->
+    kz_json:set_value(<<"pvt_port_state">>, <<"rejected">>, example_port_request()).
+
+-spec swap_comments(kz_json:object()) -> kz_json:object().
+swap_comments(Port) ->
+    %% Comment2 is the one with `waiting_for_reply=true'
+    [Comment1, Comment2] = kz_json:get_value(<<"comments">>, Port),
+    kz_json:set_value(<<"comments">>, [Comment2, Comment1], Port).
+
+-spec first_comment() -> kz_json:object().
+first_comment() ->
+    kz_json:from_list(
+        [{<<"author">>,<<"Someone">>}
+        ,{<<"content">>,<<"First comment">>}
+        ,{<<"timestamp">>,63709957526}
+        ]).
+
+-spec second_comment() -> kz_json:object().
+second_comment() ->
+    kz_json:from_list(
+        [{<<"author">>,<<"Someone">>}
+        ,{<<"content">>,<<"Second comment">>}
+        ,{<<"timestamp">>,63709957539}
+        ,{<<"waiting_for_reply">>, 'true'}
+        ]).
+
+-spec example_port_request() -> kz_json:object().
+example_port_request() ->
+    {[{<<"_attachments">>,
```

Comment:
I'm thinking can you please put this doc inside `kazoo_fixturedb/priv/dbs/....`  please?